POSHI-245

kenjiheigel opened the pull request:
https://issues.liferay.com/browse/POSHI-245

CalumR23 committed:
POSHI-245 Allow integers to be called and parsed

CalumR23 committed:
POSHI-245 Allow parsing for no quote variable calls

CalumR23 committed:
POSHI-245 Check if var should be double quoted

CalumR23 committed:
POSHI-245 Allow for parsing of non quoted variables and integers

CalumR23 committed:
POSHI-245 Change script to now test non quoted variables and integers

CalumR23 committed:
POSHI-245 Change parameters to take in non quoted variables

CalumR23 committed:
POSHI-245 Change to allow non quoted varibles in echo

CalumR23 committed:
POSHI-245 Allow for non quoted var calls in the equals statement

CalumR23 committed:
POSHI-245 Use method to check for quoted content

CalumR23 committed:
POSHI-245 Move vars and add method to check if content needs quotes

CalumR23 committed:
POSHI-245 Add new testcases for non quoted vars

CalumR23 committed:
POSHI-245 Add new testcases for non quoted vars

CalumR23 committed:
POSHI-245 Allow for non quoted parsing in contains element

CalumR23 committed:
POSHI-245 Allow for non quoted parsing in not poshi elements

CalumR23 committed:
POSHI-245 Account for non-quoted use cases

CalumR23 committed:
POSHI-245 Remove quotes on function

Commit:
LPS-165445 Replace IndexerRequestBufferExecutorWatcher with IndexerRequestBufferExecutor (because the class IndexerRequestBufferExecutor is an internal interface, and there is only one implementation, it can be directly referenced.)

CalumR23 committed:
POSHI-245 Remove quotes from resource files

Commit:
LPS-165445 Remove unused configuration

Commit:
LPS-165445 Apply

CalumR23 committed:
POSHI-245 SF

dantewang committed:
LPS-165445 SF, return value not used

Commit:
LPS-165445 SF, inline(IndexerRequestBufferExecutor cannot directly reference IndexWriterHelper, there is a circular reference problem {com.liferay.portal.search.internal.IndexWriterHelperImpl[_searchPermissionChecker] -> com.liferay.portal.search.internal.SearchPermissionCheckerImpl[_indexerRegistry] -> com.liferay.portal.search.internal.IndexerRegistryImpl[setIndexerRequestBufferOverflowHandler] -> com.liferay.portal.search.internal.buffer.DefaultIndexerRequestBufferOverflowHandler[_indexerRequestBufferExecutor] -> com.liferay.portal.search.internal.buffer.IndexerRequestBufferExecutor[indexWriterHelper] -> (circular reference)}, see f338e9e8d4ed6a08aaab001b67e0a896149d04da)

Commit:
LPS-165169 Remove unused method stream()

Commit:
LPS-169785 Clean up unnecessary spring-extender dependencies

Commit:
LPS-167050 dispatch-service - validate type after name validation (slight change required to make code clear)

Commit:
LPS-167050 dispatch-test - make sure tests use valid dispatch executor types

Commit:
LPS-167050 dispatch-service - service.xml

Commit:
LPS-167050 dispatch-service - throw exception generated by service builder

Commit:
LPS-167050 dispatch-test - test DispatchTriggerTypeException

jeffwu0724 committed:
LPS-169043 add commons-collections4 to source-formatter with version 4.4. We are using 4.2, 4.3, 4.4 in the system now, chose version 4.4, which is the highest version for better maintainance

Commit:
LPS-170042 dispatch-service - change single-node check expression to cover MEMORY-CLUSTERED and MEMORY-PERSISTED single node modes, update service to force single node memory persisted if requested mode differs from annotation given mode

Commit:
LPS-170114 Remove immediate from Portal Scheduler module

Valmir-unicap committed:
LPS-166350 New Corrected the width to 25% and added the truncate an tooltip in text and select area.

Commit:
LPS-168500 Refer to https://www.ibm.com/docs/en/db2/11.5?topic=messages-sql0000-sql0249#sql0134n, this is not permitted when using clob column in order by clause.

Commit:
LPS-167050 salesforce-connector - make sure talend executor is activated before deploy starts (this also fixes flakiness in CI environment)

Commit:
LPS-165445 Merge IndexerRequestBufferExecutor and BaseIndexerRequestBufferExecutor into DefaultIndexerRequestBufferExecutor and remove property setting as it is the only implementation

Commit:
LPS-165445 Merge DefaultIndexerRequestBufferOverflowHandler and IndexerRequestBufferOverflowHandler

Commit:
LPS-165445 Apply

Commit:
LPS-167050 validate dispatchTaskExecutorType

Commit:
LPS-169456 ObjectDefintion The PUT method should only require the same fields as the POST method

jeffwu0724 committed:
LPS-170748 move org.jfree to shared-dependencies

jeffwu0724 committed:
LPS-170748 manullay change compileInclude to compileOnly for org.jfree, since we have moved it to shared-dependencies

Commit:
LPS-165169 Simplify, remove redundant method references in class SearchRequestContributorsRegistryImpl

Commit:
LPS-165169 Inline and remove unused classes

jeffwu0724 committed:
LPS-169043 export commons-collections4 from shared-dependencies-apache-commons with version 4.4

deborasoliveira committed:
LPS-170569 Add KeywordCanDisplayInSemibold Testcase

Commit:
LPS-165445 buildLang

jeffwu0724 committed:
LPS-169043 auto sf

jeffwu0724 committed:
LPS-169043 manually change compileInclude to compileOnly for commons-collections4, because we can refer it from shared-dependencies-commons

jeffwu0724 committed:
LPS-169043 remove commons-collections4 from -includeresource in batch-engine-test, dynamic-data-lists-test and content-dashboard-test, because we can get the reference from shared-dependencies

JanisZhang committed:
LPS-170769 Remove optional<> usages in FragmentEntryProcessorContext and FragmentRendererContext

JanisZhang committed:
LPS-170769 Apply usages

JanisZhang committed:
LPS-170769 Remove optional<> usages in MenuDisplayFragmentConfiguration

JanisZhang committed:
LPS-170769 Apply usages

Commit:
LPS-170771 Convert method reference to field reference(Because property ReferencePolicyOption.GREEDY exists, I did not directly reference AMImageEntryProcessor)

liuxingye committed:
LPS-166586 Add CanFilterEntriesUsingRelationshipField test

liuxingye committed:
LPS-166589 Add CanSeeEntriesWithFilterApplied test

Commit:
LPS-170114 Remove service property and let it be implied as immediate

kenjiheigel committed:
POSHI-245 SF ignore for now

aline-doleron committed:
LPS-166350 Do not show empty tooltip

mateussandes23 committed:
LPS-170678 Create ClientExtensionElement file and CanBeCategorizedInAddWidgetList test

javalosjr96 committed:
LPS-170857 Bump google.cloud:libraries-bom to 26.1.5

BND was also modified for new imports.

kenjiheigel committed:
POSHI-245 Add poshi files for format test

kenjiheigel committed:
POSHI-245 Test for formatting

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

kenjiheigel commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-source-format/4166/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/4166/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/4166/">test-portal-source-format#4166</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/4166/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/CalumR23/liferay-portal/pull/100">CalumR23#100</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2116675916">[master] ci:test:sf - kenjiheigel &gt; CalumR23 - PR#100 - 2022-12-09[17:38:31]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/16019/">publish-testray-report#16019</a><br/></div></div>

JanisZhang committed:
LPS-170769 Remove Stream usages in BaseEditableValuesConfigurationExportImportContentProcessor

aline-doleron committed:
LPS-166350 Update snapshots

deborasoliveira committed:
LPS-170554 Add CanDisplayAllAttributes Testcase

deborasoliveira committed:
LPS-170554 Adjusting teardown for the new tests

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-source-format/3969/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/3969/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 18 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ab1e4bf5b38d0a1a06e965853308910c61d66f5f">ab1e4bf5b38d0a1a06e965853308910c61d66f5f</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/kenjiheigel/liferay-portal/tree/POSHI-245-1">POSHI-245-1</a><br/>Branch GIT ID: <a href="https://github.com/kenjiheigel/liferay-portal/commit/36aa207d66990387d84d3583652415a15b45e66c">36aa207d66990387d84d3583652415a15b45e66c</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-source-format/3969/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/3969/jenkins-report.html">here</a>.</h5><pre>     [java] java.lang.Exception: Found 7 formatting issues:
     [java] 1: ./workspaces/sample-default-workspace/client-extensions/fox-remote-app/public/index.html expected:&lt;&lt;!DOCTYPE html&gt;
     [java] [
     [java] &lt;html lang="en"&gt;
     [java] &lt;head&gt;
     [java]   &lt;meta charset="utf-8" /&gt;
     [java] 
     [java]   &lt;link href="%PUBLIC_URL%/favicon.ico" rel="icon" /&gt;
     [java] 
     [java]   &lt;meta content="width=device-width, initial-scale=1" name="viewport" /&gt;
     [java]   &lt;meta content="#000000" name="theme-color" /&gt;
     [java] 
     [java] &lt;meta content="Web site created using create-react-app" name="description" /&gt;
     [java] 
     [java]   &lt;link href="%PUBLIC_URL%/logo192.png" rel="apple-touch-icon" /&gt;
     [java]   &lt;!--
     [java]     manifest.json provides metadata used when your web app is installed on a
     [java]     user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     [java]   --&gt;
     [java]   &lt;link href="%PUBLIC_URL%/manifest.json" rel="manifest" /&gt;
     [java]   &lt;!--
     [java]     Notice the use of %PUBLIC_URL% in the tags above.
     [java]     It will be replaced with the URL of the `public` folder during the build.
     [java]     Only files inside the `public` folder can be referenced from the HTML.
     [java] 
     [java]     Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
     [java]     work correctly both with client-side routing and a non-root public URL.
     [java]     Learn how to configure a non-root public URL by running `npm run build`.
     [java]   --&gt;
     [java]   &lt;title&gt;React App&lt;/title&gt;
     [java] &lt;/head&gt;
     [java] 
     [java] &lt;body&gt;
     [java]   &lt;noscript&gt;You need to enable JavaScript to run this app.&lt;/noscript&gt;
     [java] 
     [java]   &lt;fox-remote-app route="hello-world"&gt;&lt;/fox-remote-app&gt;
     [java]   &lt;!--
     [java]     This HTML file is a template.
     [java]     If you open it directly in the browser, you will see an empty page.
     [java] 
     [java]     You can add webfonts, meta tags, or analytics to this file.
     [java]     The build step will place the bundled scripts into the &lt;body&gt; tag.
     [java] 
     [java]     To begin the development, run `npm start` or `yarn start`.
     [java]     To create a production bundle, use `npm run build` or `yarn build`.
     [java]   --&gt;
     [java] &lt;/body&gt;
     [java] &lt;/html&gt;]&gt; but was:&lt;&lt;!DOCTYPE html&gt;
     [java] [&lt;html lang="en"&gt;
     [java]   &lt;head&gt;
     [java]     &lt;meta charset="utf-8" /&gt;</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-source-format/3969/">test-portal-source-format#3969</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/3969/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/CalumR23/liferay-portal/pull/100">CalumR23#100</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2121177069">[master] ci:test:sf - kenjiheigel &gt; CalumR23 - PR#100 - 2022-12-12[09:45:28]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/14564/">publish-testray-report#14564</a><br/></div></div>

kenjiheigel commented:
ci:test:relevant:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/5718/">test-portal-acceptance-pullrequest(master)#5718</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5718/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/CalumR23/liferay-portal/pull/100">CalumR23#100</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2121649736">[master] ci:test:relevant - kenjiheigel &gt; CalumR23 - PR#100 - 2022-12-12[11:32:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/14574/">publish-testray-report#14574</a><br/></div></div>

Commit:
LPS-150203 Creating the file and adding stubs for Improve how roles are disabled when user reaches role limit

deborasoliveira committed:
LPS-170554 SF

ces-totrinh committed:
LPS-171030 Elasticsearch7 adds field mapping for userGroupRoleIds

austinchiang committed:
LPS-170179 Remove unnecessary teardown step

austinchiang committed:
LPS-170179 Split into two tests

austinchiang committed:
LPS-170179 Flag as e2e tests

austinchiang committed:
LPS-170179 Remove unnecessary test

austinchiang committed:
LPS-170179 Update test names

austinchiang committed:
LPS-170179 Remove unnecessary step

austinchiang committed:
LPS-170179 Update test

austinchiang committed:
LPS-170179 Use more efficient navigation

austinchiang committed:
LPS-170179 Rename test

austinchiang committed:
LPS-170179 Remove test covered by existing tests

austinchiang committed:
LPS-170179 Sort

sandrodw3 committed:
LPS-170968 Make close button optional

sandrodw3 committed:
LPS-170968 Add header when active item is editable

sandrodw3 committed:
LPS-170968 Restore previous panel when it's valid

sandrodw3 committed:
LPS-170968 Add comments so panel handling is more clear

sandrodw3 committed:
LPS-170968 buildLang

sandrodw3 committed:
LPS-170968 Apply secondary color to fragment comments back button

jackelyneaquino committed:
LPS-168479 Adding roles prior to permissions in order of invoke

LPS-168479 Assert Role Permission

LPS-168479 Assert role permission corrected

jeffwu0724 committed:
LPS-170748 auto generate, by running "ant build-lib-versions"

jeffwu0724 committed:
LPS-169043 auto generate, by running "ant build-lib-versions"

Commit:
LPS-150203 Add ChangesToAlreadySelectedFunctions test

Commit:
LPS-150203 Add AssertThatTheDropdownOptionsAreDisabled test

Commit:
LPS-150203 Add assertThatFunctionIsDisabled macro

Commit:
LPS-150203 Add tableChangeRole macro

Commit:
LPS-150203 Fix assertChangeRole macro

Commit:
LPS-150203 Add ROLE_CHANGE path

Commit:
LPS-150203 Add TABLE_ROLE_FIELD path

Commit:
LPS-150203 Fix ROLE path

Commit:
LPS-150203 Auto SF

Commit:
LPS-171101 Create Dispatch Trigger Impl

mateussandes23 committed:
LPS-170678 Added openEditContentPage, add FRAGMENT_SIDEBAR_COLLECTION_PANEL and FRAGMENTS_AND_WIDGETS_SEARCH_RESULTS path

JanisZhang committed:
LPS-170769 Remove Stream usages in BaseFragmentCollectionContributor

Commit:
LPS-171024 Convert method reference to ServiceTrackerMap(Because 'BulkReindexer' is not a '@ProviderType' interface, I do not directly reference the implementation class 'UserBulkReindexer')

Commit:
LPS-171024 Fix the test

ricardocousolr committed:
LPS-168169 Lower numerical value of priority means higher precendence

Entries with no priority (null) before are set to -1 in step v1_7_0, so
they have the most precendence. To respect this we must order by
priority asc too. Entries with the same priority value are sorted by
createDate (more recent means more precedence).

sandrodw3 committed:
LPS-171029 Render header even if there's no any panel

sandrodw3 committed:
LPS-171029 Add tests ItemConfiguration component

Commit:
LPS-150203 Add workaround

Commit:
LPS-149027 Creating the file and adding stubs for DXP Cloud activation form already submitted scenario

Commit:
LPS-149027 Add CanViewAUsefulMessage test

Commit:
LPS-149027 Auto SF

achaparro committed:
LPS-168721 Always show a message in the Upgrade Report

Commit:
LPS-171101 Update Object Definition

Commit:
LPS-171113 Call Dispatch Trigger after Create the Task

Commit:
LPS-171112 Update Status

Commit:
LPS-171112 SF

Commit:
LPS-171112 Enforce Type

deborasoliveira committed:
LPS-170569 Add ClayAlertToast path file

deborasoliveira committed:
LPS-170569 Adjusting the tasks of KeywordCanDisplayInSemibold Testcase

deborasoliveira committed:
LPS-170554 Adjusting the tasks of CanDisplayAllAttributes Testcase

deborasoliveira committed:
LPS-170554 SF

deborasoliveira committed:
LPS-170554 Add ClayAlert.path file for the static alert paths

WYZ-Issac committed:
LPS-167926 Add a test to ensure the same friendly url can re-add for different languages in a layout

WYZ-Issac committed:
LPS-167926 Add a test to ensure the same friendly url can re-add for different languages in web content

andrea-ale-sbarra committed:
COMMERCE-10471 changed cookie type

evanilson-santana committed:
LPS-168858 Update groovy script with new object id name

evanilson-santana committed:
LPS-168858 add an expression on the edit task to validate the action was edited

danvb112 committed:
LPS-171289 Fix parameter validation in Object Relationship using parameterObjectFieldName instead of ID

patriciajeanperez committed:
LPS-164328 Remove search step in User macro

julschong committed:
LPS-171276 Remove Stream usages and use for loop instead

julschong committed:
LPS-171276 Auto, SF

tinatian committed:
LPS-165169 SF, simplify

dantewang committed:
LPS-170769 SF

shuyangzhou committed:
LPS-171360 Remove NotificationEventFactory, just create NotificationEvent directly.

shuyangzhou committed:
LPS-171360 Remove notifications channel infrastructure which is no longer used anywhere.

shuyangzhou committed:
LPS-171360 Semantic versioning

shuyangzhou committed:
LPS-171357 Avoid using StoreFactory from component.

shuyangzhou committed:
LPS-171357 Extract ConvertProcess as components

shuyangzhou committed:
LPS-171357 Explicitly checking store readiness

shuyangzhou committed:
LPS-171357 Merge StoreFactory into DLStoreImpl

shuyangzhou committed:
LPS-171357 Semantic versioning

Commit:
LPS-167050 dispatch - Autogenerated

Commit:
LPS-167050 dispatch - Autogenerated - baseline

Commit:
LPS-171323 Pagination fix

Commit:
LPS-171324 Add Styles To Dashboard Charts

Commit:
LPS-170405 Fix test SystemObjectAccount#CanAssociateAnyTypeOfAccount

evanilson-santana committed:
LPS-169427 Update ERC field validation from integer to string on the expression builder

shuyangzhou committed:
LPS-171357 Fix test

epark18 committed:
LRQA-73002 Rename test for duplicate

CalumR23 committed:
POSHI-459 Generate CHANGELOG

CalumR23 committed:
POSHI-459 Update default poshi runner version

CalumR23 committed:
POSHI-459 Use latest gradle-plugins-poshi-runner

CalumR23 committed:
POSHI-459 Update poshi-core version

Commit:
LPS-170678 SF

Commit:
LPS-168898 SF

interaminense committed:
LRAC-12601 Fix commerce channels total when toggle commerce sync

itstali committed:
LPS-171339 Fix RaylifeHomePage#ZipCodeIsRequired Test

Commit:
LRAC-12627 Modify the branch that is used in AC acceptance tests

danielCamara02 committed:
LPS-171134 Add CanApprovePageInAPublication test

interaminense committed:
LRAC-12601 Do no use uid because ids are generated every time table is rendered

interaminense committed:
LRAC-12601 Update generated types

kiwm committed:
LPS-171345 Change the display logic on Attachment.tsx function File

rbohl committed:
LPS-166942 wordsithing

Commit:
LPS-170826 Add Revalidate function

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

Commit:
LPS-170826 Add Rest Implementations for Task, Subtask, CaseResult and TaskUsers

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

Commit:
LPS-170826 Simplify

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

Commit:
LPS-170826 Update Outlet parameters and standardize

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

Commit:
LPS-170826 Use Outlet  with new properties on Testflow and Subtask

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

Commit:
LPS-170826 Update Outlet Router

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

tinatian committed:
LPS-165445 Baseline

pei-jung committed:
LPS-171336 Implements Clusterable so that it will run on every node in cluster

ealonso committed:
LPS-171352 As message said, we need to keep this line here

liferay-translation committed:
LPS-77699 Update Translations

david-truong committed:
LPS-164162 fix css styles not applied on content pages in a publication

ealonso committed:
LPS-169585 Removes no necessary custom css

ealonso committed:
LPS-169585 It should be namespace

ealonso committed:
LPS-169585 Uses taglib

ealonso committed:
LPS-169569 Removes no needed custom css and classes

ealonso committed:
LPS-169569 SF

ling-alan-huang committed:
LPS-168928 Auto SF

ealonso committed:
LPS-169576 Removes unnecessary css and classes

ealonso committed:
LPS-169576 SF

ealonso committed:
LPS-169576 SF - simplify

ealonso committed:
LPS-169598 Removes unnecessary css and classes

ealonso committed:
LPS-169596 Removes unnecessary css

ealonso committed:
LPS-169529 Update classes and removes unnecessary css file

Commit:
LRQA-78859 Fix typo

JanisZhang committed:
LPS-170769 semver

brianchandotcom committed:
artifact:ignore image-uploader-web 5.0.30 prep next

brianchandotcom committed:
artifact:ignore plugins-admin-web 5.0.25 prep next

brianchandotcom committed:
artifact:ignore wiki-web 7.0.73 prep next

brianchandotcom committed:
artifact:ignore portal-settings-web 5.0.38 prep next

brianchandotcom committed:
artifact:ignore expando-taglib 5.0.19 prep next

brianchandotcom committed:
artifact:ignore message-boards-web 5.0.68 prep next

brianchandotcom committed:
artifact:ignore blogs-web 6.0.78 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-lists-web 6.0.59 prep next

brianchandotcom committed:
artifact:ignore document-library-web 6.0.123 prep next

brianchandotcom committed:
artifact:ignore knowledge-base-web 5.0.73 prep next

brianchandotcom committed:
artifact:ignore site-admin-web 5.0.71 prep next

brianchandotcom committed:
artifact:ignore account-admin-web 2.0.72 prep next

brianchandotcom committed:
artifact:ignore users-admin-web 6.0.80 prep next

brianchandotcom committed:
artifact:ignore commerce-order-web 5.0.69 prep next

brianchandotcom committed:
artifact:ignore commerce-pricing-web 4.0.65 prep next

brianchandotcom committed:
artifact:ignore commerce-product-definitions-web 7.0.82 prep next

brianchandotcom committed:
artifact:ignore commerce-product-options-web 4.0.46 prep next

brianchandotcom committed:
artifact:ignore commerce-shipment-web 4.0.39 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-web 5.0.71 prep next

brianchandotcom committed:
artifact:ignore roles-admin-web 5.0.63 prep next

brianchandotcom committed:
artifact:ignore calendar-web 5.0.62 prep next

brianchandotcom committed:
artifact:ignore site-navigation-admin-web 4.0.59 prep next

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.115 prep next

brianchandotcom committed:
artifact:ignore user-groups-admin-web 5.0.38 prep next

brianchandotcom committed:
artifact:ignore journal-web 5.0.112 prep next

Commit:
LPS-171366 Fix typo

brianchandotcom committed:
LPS-171276 SF

brianchandotcom committed:
artifact:ignore feature-flag-web 1.0.2 prep next

sandrodw3 committed:
LPS-170872 Sort

4lejandrito committed:
LPS-169781 Using local services

As in f6187cd26a98099ba86c801436cfa7eb0af961df that fixed this same
problem in the old dto converter.

victorg1991 committed:
LPS-171107 Remove unused role

brianchandotcom committed:
LPS-170769 SF

brianchandotcom committed:
artifact:ignore frontend-taglib-util-freemarker-contributor 5.0.12 prep next

pedro-oliveira446 committed:
LPS-170932 add new function to extract the requestId from url

Commit:
LPS-169892 Add CanOrderCategoryFacetTerms testcase

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
Baseline

brianchandotcom committed:
LPS-168858 #sfme

brookedalton committed:
LPS-171134 Move to publications workflow testcase file

Commit:
LPS-171227 Add Feature Flag to LPS-165850

patriciajeanperez committed:
Revert "LPS-170474 SF"

This reverts commit 27115da6a995b20d3b032aa2379d0ba4379eef42.

patriciajeanperez committed:
Revert "LPS-170474 Restore PrincipalThreadLocal to original state"

This reverts commit f6e6919430406792fc94e5a1fb0264726c241b7d.

Commit:
LPS-171227 afcb

brianchandotcom committed:
LPS-167050 SF

brianchandotcom committed:
LPS-170967 Wordsmith

brianchandotcom committed:
LPS-170967 Regen

kenjiheigel committed:
POSHI-245 Fix tests

rafaprax committed:
LPS-171442 Make Opcodes.V1_6 the default option instead of the Opcodes.V1_2 and patch mvel2

rafaprax committed:
LPS-171442 Update usages

rafaprax committed:
LPS-171442 ant build-lib-versions

john-co committed:
LPS-170569 Refactor test

john-co committed:
LPS-170569 Remove duplicate path

kenjiheigel committed:
POSHI-245 Account for function usages

kenjiheigel committed:
POSHI-245 Account for 'return' element values

kenjiheigel committed:
POSHI-245 Fix tests

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-171442 prep next

kenjiheigel committed:
POSHI-245 Add additional test cases
